libraries/core_libs/consensus/include/pbft/sync_queue.hpp

```diff
@@ -14,8 +14,9 @@ class SyncBlockQueue {
  public:
   SyncBlockQueue() = default;
 
-  bool push(SyncBlock &&block, dev::p2p::NodeID const &node_id, uint64_t max_pbft_size);
-  std::pair<SyncBlock, dev::p2p::NodeID> pop();
+  bool push(SyncBlock &&block, dev::p2p::NodeID const &node_id, uint64_t max_pbft_size,
+            std::vector<std::shared_ptr<Vote>> = {});
```

Comment:
std::vector<std::shared_ptr<Vote>> &&

Reply:
where is the argument name ?

Reply:
From Miho branch, has fixed after rebase